modules/javafx.controls/src/main/java/javafx/scene/control/TreeTablePosition.java

```diff
@@ -77,6 +77,16 @@ public TreeTablePosition(@NamedArg("treeTableView") TreeTableView<S> treeTableVi
         nonFixedColumnIndex = treeTableView == null || tableColumn == null ? -1 : treeTableView.getVisibleLeafIndex(tableColumn);
     }
 
```

Comment:
Suggest to add 
    // Not public API
before this description.

(Similar to the constructor above this newly added constructor)

Reply:
Corrected in the next commit.